pkg/querier/api/consistency.go

```diff
@@ -223,11 +222,6 @@ func EncodeOffsets(offsets map[int32]int64) EncodedOffsets {
 	// re-allocations because we under-counted digits.
 	size := versionLen
 	for partitionID, offset := range offsets {
-		// Skip empty partitions.
```

Comment:
Note to reviewers: this was a trick used to only restrict the propagated offsets to the in-use partitions. We don't need this anymore, because now we fetch only the offsets for in-use partitions.